### PR TITLE
NER-436

### PR DESCRIPTION
**Pull request by krahser**

### CODE REVIEW

File: common/head_tag.html [hunk 1]
```diff
@@ -62,10 +62,19 @@
         `);
         $headerMenu.append($homeMenuItem);
 
+        $progressMenuItem = $(`
+            <li>
+                <a href="${base_url}/my-progress/" class="nav-li-profile">`
+                    currentMenu[1] + `
+                </a>
```
Comment (marcokuchlaopenenglish) [inline common/head_tag.html:69]: Is the styling right here? On the others we have the `<i class...` tag
Also the class on the link is using `nav-li-profile`

**Review by marcokuchlaopenenglish — COMMENTED** [review]
Besides one comment, LGTM

File: common/head_tag.html (hunk 1 above)
Comment (krahser) [reply common/head_tag.html:69]: right, but if we look at the [new header](https://app.zeplin.io/project/605a6e59dc29b7078652e6f3/screen/60e4663d3f93c6b9145504fa) no one is having icons.

Comment (marcokuchlaopenenglish) [reply common/head_tag.html:69]: Oh got it, sounds good then

**Review by marcokuchlaopenenglish — APPROVED** [review]

**Comment by marcokuchlaopenenglish** [comment]
One question though, updating this we will be changing the current header for all students? I mean, should we consider using a flag for this too?

**Pull request merged**